integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMultiDomainModelsWithLoadBalancer.java

```diff
@@ -552,7 +552,7 @@ void testLivenessProbe() {
 
   /**
    * Verify dataHome override in a domain with domain in image type.
-   * In this domain, set dataHome to /u01/oracle/mydata in domain custom resource
+   * In this domain, set dataHome to /u01/mydata in domain custom resource
```

Comment:
This is dataHome override tests. The dataHome is set to /u01/oracle/mydata on purpose. The test will fail if it is set to /u01/mydata.